### PR TITLE
First version of steps to create vanilla design specs

### LINKED ISSUE #28: Document how to create a design spec
We need to create a document which details how to create a design spec.

https://github.com/vanilla-framework/vanilla-framework/issues/1113

### PR DESCRIPTION
**Pull request by yaili**
Outline of the structure of the Vanilla design specs and steps to create new specs

Fixes #28

### CODE REVIEW

File: design/vanilla-design-specs.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Detailed written spec
+
+The detailed spec file should include:
+
+- Title of the pattern: e.g. "[Vanilla: Buttons](https://github.com/ubuntudesign/vanilla-design/blob/master/Buttons/buttons.md)" or "[Brochure theme: Buttons](https://github.com/ubuntudesign/vanilla-brochure-theme-design/blob/master/Buttons/buttons.md)"
+- All the properties needed to build the pattern: e.g. font size, text colour, padding, borders, shadows, backgrounds 
+- Where applicable, include responsive variations for different viewports
+- Where applicable, include: hover, active and focused states
+
+The spec shouldn't be written in CSS format. For example, avoid writing "font-size", write "font size" instead.
```
Comment (nottrobin) [inline design/vanilla-design-specs.md]: :+1: :heart:

**Comment by nottrobin** [comment]
Great work @yaili thanks. :+1: from me.

**Comment by anthonydillon** [comment]
:+1: Looks great.

**Comment by yaili** [comment]
@anthonydillon what do I do now? squash and merge?

**Comment by anthonydillon** [comment]
@yaili no, you still need to wait until the PR is 24 hours old.

**Comment by nottrobin** [comment]
@yaili do give others an opportunity to object if they want.

**Comment by yaili** [comment]
Got it. 👍

**Pull request merged**